muikku-core-plugins/src/main/resources/META-INF/resources/scripts/gui/muikku-material-loader.js

```diff
@@ -530,6 +530,29 @@
           memoFieldElement.addClass('ckeditor-field');
         }
         $(object).replaceWith(memoFieldElement);
+        
+        // #3120 memo word counter (non-richedit)
+        if (data.meta.richedit != true) {
+          var wordCountContainer = $('<div class="word-count-container">')
+            .append($('<span class="word-count-title">').text(getLocaleText('plugin.workspace.memoField.wordCount')))
+            .append('<span class="word-count">');
+          memoFieldElement.after(wordCountContainer);
+          var countMethod = function() {
+            var text = memoFieldElement.val().trim();
+            $(wordCountContainer).find('.word-count').text(text == '' ? 0 : text.split(/\s+/).length);
+          };
+          var timer = undefined;
+          memoFieldElement.on('focus', $.proxy(function() {
+            timer = setInterval($.proxy(function() {
+              countMethod();
+            }, this), 1000);
+          }, this));
+          memoFieldElement.on('blur', $.proxy(function() {
+            countMethod();
+            clearInterval(timer);
```

Comment:
If the element is initially on focus, blur will trigger `clearInterval(undefined)`. I'd initialize it with null and null check it before clearing.